frontend/modules/recipe/components/Recipe.js

```diff
@@ -172,7 +193,7 @@ var RecipeScheme = injectIntl(React.createClass({
                 { (this.props.data.source) ?
                   <div>{ formatMessage(messages.source) }: { this.props.data.source }</div>
                   :
-                  <div>{ formatMessage(messages.created_by) }: { this.props.data.username }</div>
+                  <div>{ formatMessage(messages.created_by) }: { this.props.data.username } { this.showEditLink() }</div>
```

Comment:
Ah, I totally misunderstood what source was then. On a related note, why show the source instead of the user? Wouldn't you want to show both anyway?